test/e2e/create_cluster_test.go

```diff
@@ -154,3 +154,51 @@ func TestNoneCreateCluster(t *testing.T) {
 	// etcd restarts for me once always and apiserver two times before running stable
 	// e2eutil.EnsureNoCrashingPods(t, ctx, client, hostedCluster)
 }
+
+// TestAgentCreateCluster implements a test that mimics the operation described in the
+// HyperShift quick start (creating a basic guest cluster).
+//
+// This test is meant to provide a first, fast signal to detect regression; it
+// is recommended to use it as a PR blocker test.
+func TestAgentCreateCluster(t *testing.T) {
+	ctx, cancel := context.WithTimeout(testContext, 35*time.Minute)
+	defer cancel()
+
+	t.Parallel()
+	g := NewWithT(t)
+	client, err := e2eutil.GetClient()
+	g.Expect(err).NotTo(HaveOccurred(), "failed to get k8s client")
+
+	clusterOpts := globalOpts.DefaultClusterOptions()
+	clusterOpts.Name = "hypershift"
+	hostedCluster := e2eutil.CreateCluster(t, ctx, client, &clusterOpts, hyperv1.AgentPlatform, globalOpts.ArtifactDir)
+
+	// Get the newly created nodepool
+	nodepool := &hyperv1.NodePool{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: hostedCluster.Namespace,
+			Name:      hostedCluster.Name,
```

Comment:
why does this test have to be different than the existing TestCreateCluster?

Reply:
I guess I can remove the agent specific validations and just use the existing TestCreateCluster